chromium_src/components/permissions/features.cc

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
These definitely should not go in chromim_src. This is only for overrides of existing files. You can make it available to other chromium_src files (like permission_prompt_bubble_view.cc above) by adding a dep in brave/browser:browser_process